### DISCUSSION TITLE
Extra Networks "Description [Replace]" buttons do nothing?

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#1009] [Q&A] [answered]
What do the "Description [Replace]" buttons do?
Nothing seems to change when I click them.

**Comment by vladmandic** [comment, marked as answer]
Just above the list you have a text field you can enter anything. And when you click replace that text is written to txt file next to model and can be read using read description.

**Reply by brknsoul** [reply]
Ahh, I'm blind lol!

Thanks!
